### PR TITLE
(#1151) qulice update to fix build errors

### PR DESCRIPTION
**Pull request by fabriciofx**
For #1151 

- qulice update to fix build errors
- leave a puzzle to fix all files don't pass in the updated qulice
- we should fix them and remove from `pom.xml`

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1198](https://github.com/yegor256/cactoos/issues/1198) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1198](https://github.com/yegor256/cactoos/issues/1198) is assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/C63314D6Z/0cfd8e21-26ed-4b50-8408-a342a667cc74); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by fabriciofx** [comment]
@victornoel ping

**Comment by victornoel** [comment]
@fabriciofx good approach
@paulodamaso it's good to merge as-is

**Comment by fabriciofx** [comment]
@victornoel thanks!

File: pom.xml [hunk 1]
```diff
@@ -158,10 +158,108 @@ The MIT License (MIT)
           <plugin>
             <groupId>com.qulice</groupId>
             <artifactId>qulice-maven-plugin</artifactId>
-            <version>0.17.3</version>
+            <version>0.18.19</version>
             <configuration>
               <excludes combine.children="append">
                 <exclude>checkstyle:/src/site/resources/.*</exclude>
+                <!--
+                 @todo #1151:30min To fix qulice build erros is necessary update it.
```
Comment (paulodamaso) [inline pom.xml]: @fabriciofx Is necessary to update qulice again? Or updating from 0.17.3 to 0.18.19 already fixed the problems? Puzzle text is not clear

Comment (fabriciofx) [reply pom.xml]: @paulodamaso I'll fix it.

**Comment by fabriciofx** [comment]
@paulodamaso done. Please, check it out.

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @victornoel[/z](https://www.0crat.com/u/victornoel), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1198](https://github.com/yegor256/cactoos/issues/1198) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by sereshqua** [comment]
@0crat quality bad

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31)

**Comment by 0crat** [comment]
Quality review completed: +4 point(s) just awarded to @sereshqua[/z](https://www.0crat.com/u/sereshqua)

**Pull request merged**